tools/ports/sdl_ttf.py

```diff
@@ -0,0 +1,43 @@
+import os, shutil, logging
+
+TAG = 'version_1'
+
+def get(ports, settings, shared):
+  if settings.USE_SDL_TTF == 2:
+    sdl_build = os.path.join(ports.get_build_dir(), 'sdl2')
+    freetype_build = os.path.join(ports.get_build_dir(), 'freetype')
+    assert os.path.exists(sdl_build), 'You must use SDL2 to use SDL2_ttf'
+    assert os.path.exists(freetype_build), 'You must use Freetype to use SDL2_ttf'
```

Comment:
we should just automatically enable freetype. see `process_dependencies()` in `libpng.py` for the libpng port, for an example, it requires zlib. note also how it uses `'-s', 'USE_ZLIB=1'` when building the libpng library.